### DISCUSSION TITLE
Building a Digital Twin of the iCub Humanoid Robot’s Head

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#611] [Research news]
Hi @robotology/everyone 👋🏻 

We, at [iCub Tech][icub-tech], are thrilled to share with you that we've just published a technical article with the [MathWorks][mathworks]® about how we've been working to build **digital twins** of our mechanisms to improve control performance.

Check it out
- 📄 "[Building a Digital Twin of the iCub Humanoid Robot’s Head][article]"

https://user-images.githubusercontent.com/3738070/224051571-c62f6336-3511-4d4d-ad48-624260422717.mp4

Hope you'll enjoy reading it up!

cc @mfussi66 @salvi-mattia @Mick3Lozzo @maggia80 

[icub-tech]: https://github.com/icub-tech-iit
[mathworks]: https://www.mathworks.com
[article]: https://www.mathworks.com/company/newsletters/articles/building-a-digital-twin-of-the-icub-humanoid-robots-head.html

**Comment by alecive** [comment]
Wow, pretty impressive work!

**Reply by pattacini** [reply]
Thanks 😃

**Comment by towardthesea** [comment]
Wow, it is cool approach.

--
NGUYEN Dong Hai Phuong
Application lead
NEURA Robotics GmbH

**Reply by pattacini** [reply]
Thanks @towardthesea 👍🏻 
Hope you're doing great!
